frontend/src/components/agency/configure-connector-modal/ConfigureConnectorModal.jsx

```diff
@@ -191,7 +192,10 @@ function ConfigureConnectorModal({
               <ManageFiles selectedItem={connectorId} />
             )}
             {activeKey === "MANUALREVIEW" && (
-              <DBRules connDetails={connDetails} />
+              <DBRules
+                connDetails={connDetails}
+                workflowDetails={workflowDetails}
+              />
```

Comment:
```suggestion
              {DBRules &&<DBRules
                connDetails={connDetails}
                workflowDetails={workflowDetails}
              />}
```

This way it won't cause any breaking if component is not imported properly